response_operations_ui/controllers/party_controller.py

```diff
@@ -126,22 +126,22 @@ def search_respondent_by_email(email):
 
 
 def update_contact_details(ru_ref, respondent_id, form):
+    logger.debug('Updating respondent details', respondent_id=respondent_id)
 
     new_contact_details = {
-        "first_name": form.get('first_name'),
-        "last_name": form.get('last_name'),
+        "firstName": form.get('first_name'),
+        "lastName": form.get('last_name'),
         "email_address": form.get('hidden_email'),
         "new_email_address": form.get('email'),
-        "telephone": form.get('telephone'),
-        "respondent_id": respondent_id
+        "telephone": form.get('telephone')
     }
 
     old_contact_details = get_respondent_by_party_id(respondent_id)
     contact_details_changed = _compare_contact_details(new_contact_details, old_contact_details)
 
     if len(contact_details_changed) > 0:
-        url = f'{app.config["BACKSTAGE_API_URL"]}/v1/party/update-respondent-details/{respondent_id}'
-        response = requests.put(url, json=new_contact_details)
+        url = f'{app.config["PARTY_URL"]}/party-api/v1/respondents/id/{respondent_id}'
```

Comment:
This isn't the correct URL for update contact details on party or are we changing it to be more restful? This is: /respondents/change_respondent_details/<respondent_id>

Reply:
Copied from the [call in backstage](https://github.com/ONSdigital/ras-backstage/blob/a173984dddb466210df500046f4e7b11ec4bb679/ras_backstage/controllers/party_controller.py#L69) which points at [the respondent view in party](https://github.com/ONSdigital/ras-party/blob/acb01273d3a5c001be75e48a9f2257304ed550cf/ras_party/views/respondent_view.py#L54). It might be that the [API documentation](https://github.com/ONSdigital/ras-party/blob/master/API.md#example-json-data-for-put-1) is out of date?

Reply:
@gemmai95 ^ ICYMI.

Reply:
Yeah the API.md for party is out of date.